src/main/webapp/web.xml

```diff
@@ -8,8 +8,85 @@
         <servlet-name> index </servlet-name>
         <servlet-class>mate.hwdao.controllers.IndexController</servlet-class>
     </servlet>
-<servlet-mapping>
-    <servlet-name>index</servlet-name>
-    <url-pattern>/index</url-pattern>
-</servlet-mapping>
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name> allDrivers</servlet-name>
+        <servlet-class>mate.hwdao.controllers.DisplayAllDriversController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name> allDrivers</servlet-name>
+        <url-pattern>/drivers/</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name> addDriver </servlet-name>
+        <servlet-class>mate.hwdao.controllers.AddDriverController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>addDriver</servlet-name>
+        <url-pattern>/drivers/add</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name> addManufacturer </servlet-name>
+        <servlet-class>mate.hwdao.controllers.AddManufacturerController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>addManufacturer</servlet-name>
+        <url-pattern>/manufacturer/add</url-pattern>
+    </servlet-mapping>
+    <servlet>
+        <servlet-name> addCars </servlet-name>
+        <servlet-class>mate.hwdao.controllers.AddCarController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>addCars</servlet-name>
+        <url-pattern>/cars/add</url-pattern>
+    </servlet-mapping>
+    <servlet>
+        <servlet-name> addDriverToCar </servlet-name>
+        <servlet-class>mate.hwdao.controllers.AddDriverToCarController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>addDriverToCar</servlet-name>
+        <url-pattern>/cars/addDriverToCar</url-pattern>
```

Comment:
Good URLs naming in your project:
/cars/drivers/add - (add driver to car)

Reply:
done